src/App.tsx

```diff
@@ -77,40 +79,57 @@ function App() {
     handleResize();
   }, [dispatch])
 
-  // ROUTING TABS
+  const listenForUrlLanguage = (pathname: string) => {
+    if (pathname.includes("/en/") && language !== LanguageType.english) {
+      dispatch({
+        type: "SELECT_LANGUAGE",
+        payload: LanguageType.english
+      })
+    }
+    else if (pathname.includes("/fr/") && language !== LanguageType.french) {
+      dispatch({
+        type: "SELECT_LANGUAGE",
+        payload: LanguageType.french
+      })
+    }
+  }
+
+  const history = useHistory()
+  listenForUrlLanguage(history.location.pathname)
+
   return (
     <div className="App">
       <NavBar />
       <CookieBanner />
-      <Switch >
-        <Route path="/About">
+      <Switch>
+        <Route path="/:language/About">
           <About />
         </Route>
-        <Route path="/Explore">
+        <Route path="/:language/Explore">
           <Explore />
         </Route>
-        <Route path="/Dashboard">
-          <Redirect to="/Explore" />
+        <Route path="/:language/Dashboard">
```

Comment:
We should make any of our old routes route to our new english routes, so that people who have serotracker urls saved on their browser don't visit a url that no longer exists

e.g. /Dashboard --> /en/Dashboard